src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -208,6 +217,27 @@ CHIP_ERROR pychip_DeviceController_ConnectBLE(chip::Controller::DeviceCommission
                                    .SetDiscriminator(discriminator));
 }
 
+CHIP_ERROR pychip_DeviceController_ParseQRCode(const char* qrCode, SetupPayload* output)
```

Comment:
Have you seen `chip/setup_payload/setup_payload.py`? It stores CHIP SetupPayload attibutes in a dictionary, but perhaps we should unify these two Python classes at some point.

Reply:
Hadn't seen this go by. I changed my python code to use that. Right now, it doesn't actually create a dictionary, it creates a list of pairs, so I changed that in the next commit - I hope that's ok.